src/lib/utils/thorchain/memo/addAggregatorAddressToMemo.ts

```diff
@@ -0,0 +1,60 @@
+import { MEMO_PART_DELIMITER } from './constants'
+
+type AddShortenedAggregatorToMemoArgs = {
```

Comment:
two nitpicks in one: use address terminology and don't mention shortening since this isn't the concern of this util - it's only adding the aggregator address and doesn't have knowledge about shortening

```suggestion
type AddShortenedAggregatorAddressToMemoArgs = {
```

and even better:

```suggestion
type AddAggregatorAddressToMemoArgs = {
```

Reply:
in [2a6fc6a](https://github.com/shapeshift/web/pull/7146/commits/2a6fc6a57521f17fb2fa9ee67d3d46f222c6e53c)